src/blog/2024/05/how-to-setup-node-red-on-raspberry-pi.md

```diff
@@ -0,0 +1,237 @@
+
+For more information on the audit log, refer to [Audit log](/docs/user/logs/#audit-log).
+
+For enhanced security and granular control over team access, refer to [Role-Based Access for your Node-RED applications](/blog/2024/04/role-based-access-control-rbac-for-node-red-with-flowfuse/).
+
+To learn more about security provided by FlowFuse at every level, refer to the [Security Statement](/product/security/#application).
+
+## Integrating sensors and actuators
+
+Interacting with sensors and actuators with Raspberry Pi can be complex, which often involves Python programming and managing various libraries. However, with FlowFuse running directly on your hardware device, the process becomes simpler and faster.
+
+There are numerous custom nodes available for interacting with both sensors and actuators in Node-RED. One of the most popular nodes for controlling actuators is [node-red-node-pi-gpio](https://flows.nodered.org/node/node-red-node-pi-gpio). This node allows you to send on/off commands to actuators and perform Pulse Width Modulation (PWM), useful for controlling motor speed, LED brightness, servo motor positions and more.
+
+For reading data from sensors, you can search for nodes by sensor names in the palette manager. You'll see several nodes as search results. There are nodes available for almost all types of sensors in Node-RED. Select the appropriate node by reviewing its descriptions and README. Alternatively, seek assistance on the [Node-RED Forums](https://discourse.nodered.org/#!). 
+
+### Interfacing DHT11 sensor to Raspberry Pi
+
+To demonstrate the integration of a sensor with Node-RED, we will walk you through the steps of connecting a DHT11 sensor to read temperature data with Node-RED.
+
+1. Grab three **female-to-female** jumper wires.
+2. Connect the wire from the signal(s or +) pin of the DHT11 sensor to the **GPIO 4** pin on the Raspberry Pi.
+3. Connect the wire from the Vcc(v or out) pin of the DHT11 sensor to any **5-volt** pin on the Raspberry Pi.
+4. Connect the wire from the Ground(g or -) pin of the DHT11 sensor to any **ground (GND)** pin on the Raspberry Pi.
+
+If you are unsure about the pin configurations, you can refer to the images below:
+
+!["Image showing the Raspberry Pi pins with diagram"](./images/how-to-setup-node-red-on-raspberry-pi-pins.png "Image showing the Raspberry Pi pins with diagram"){data-zoomable}
+
+### Reading sensor data with Node-RED
+
+#### Installing bcm2835 library 
+
+The bcm2835 library is essential for Raspberry Pi (RPi) as it provides access to GPIO and other IO functions on the Broadcom BCM 2835 chip, used in the Raspberry Pi 4. This allows control and interface with various external devices connected to the GPIO pins on the RPi 4 boards.
```

Comment:
What happens on the RPI5?

Reply:
This dht sensor node requires the bcm2835 library to be preinstalled, which only works on Raspberry Pi models up to version 4